### DISCUSSION TITLE
Get a blank page in FastAPI, the code works fine on other machines. changing python version does not affect?

### DISCUSSION THREAD

**Discussion by agoor97** [fastapi/fastapi#9645] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI

app = FastAPI()


@app.get("/items")
async def read_items():
    return [{"name": "Foo"}]
```

### Description

I get a blank page 

### Operating System

Windows

### FastAPI Version

0.95.1

### Python Version

3.10.9

**Comment by jgould22** [comment]
You need to provide more context otherwise it makes it very difficult for people to help.

Some things that come to mind
> I get a blank page

A blank page in what? 

A browser? 

What does the network tab in the developer tools tell you? 

What does the result look like if you call the endpoint with Curl?
```
curl localhost:5000/items/
```

Are you using a virtual environment (or something like docker) so you know the dependencies are correct?

Do you see the connection logged by uvicorn when you make the request?

Do you have any logging in your endpoint that you might use to log in the controller?

**Comment by agoor97** [comment]
I get only a blank page (no docker) running on localhost:8000/docs.
![2023-06-10_002031](https://github.com/tiangolo/fastapi/assets/81787449/9c683fde-a8ae-40af-bb7d-fac43c07f216)
![2023-06-10_002109](https://github.com/tiangolo/fastapi/assets/81787449/d386ac59-b0f4-4534-8711-35767c1cebbb)
![2023-06-10_002139](https://github.com/tiangolo/fastapi/assets/81787449/02c0f2f9-7113-45bd-99da-8fd70feeec9c)

The code works fine on other machines, I tried different versions of python. different conda environments, but the same problem

**Comment by jgould22** [comment, marked as answer]
Ok so a blank page for the docs.

FastAPI Serves the Javascript for the docs from a CDN. It looks like your PC may not be able to load the javascript.

You can open the network panel in chrome and see if the request is being blocked.

**Reply by Sierra-034** [reply]
Is there anything I can do to overcome this issue? or is it a temporary thing?

**Reply by jgould22** [reply]
It depends on why you machine is unable to connect to the CDN.

You can over come it by hosting the JS yourself locally so the client doesnt need to load from a CDN.
